### PR TITLE
wip: Travis CI support

### PR DESCRIPTION
**Pull request by cromwellryan**
- [x] Include Gemfile so Travis installs the necessary test gems (minitest)
- [x] .travis.yml (if necessary)
- [x] Build badge

relates #10

### CODE REVIEW

**Comment by agriffis** [comment]
Thanks! Any thoughts on why the build is failing? It runs locally for me with just `rake` (which is different from Travis running with `bundle exec rake`)

```
$ rake
Ignoring ffi-1.9.14 because its extensions are not built.  Try: gem pristine ffi --version 1.9.14
Ignoring json-2.0.1 because its extensions are not built.  Try: gem pristine json --version 2.0.1
/usr/bin/ruby test/rubypants_test.rb
Ignoring ffi-1.9.14 because its extensions are not built.  Try: gem pristine ffi --version 1.9.14
Ignoring json-2.0.1 because its extensions are not built.  Try: gem pristine json --version 2.0.1
Run options: --seed 33836

# Running:

...........

Finished in 0.034897s, 315.2169 runs/s, 3753.9473 assertions/s.

11 runs, 131 assertions, 0 failures, 0 errors, 0 skips
```

**Comment by agriffis** [comment]
Hmm, it seems to be because `lib/rubypants/version.rb` has

```
class RubyPants
```

and `lib/rubypants/core.rb` has

```
class RubyPants < String
```

and when running with `bundle exec rake` then these conflict. I don't know the best practice for resolving this, though.

File: README.rdoc [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 {<img src="https://img.shields.io/gem/v/rubypants.svg?style=plastic">}[https://rubygems.org/gems/rubypants]
 
+{<img src="https://travis-ci.org/jmcnevin/rubypants.svg?branch=master" alt="Build Status" />}[https://travis-ci.org/jmcnevin/rubypants]
```
Comment (agriffis) [inline README.rdoc]: Let's use the shields.io badge with `?style=plastic` so it matches the other one, and get them to render on the same line (however that works in rdoc)

Comment (cromwellryan) [inline README.rdoc]: Totes!

**Comment by cromwellryan** [comment]
RE `class RubyPants < String`
I'll take a look and submit back. If it's impactful and something you'd rather do in another PR, we can override the test command for Travis in the `.travis.yml` to just use `rake` vs `bundle exec rake`.

**Comment by cromwellryan** [comment]
At first glance, I think this might be outside the realm of this particular PR.

The common pattern would be to use a top level `module` of `RubyPants` which would expose the version.  Here are a few examples:

* [flip](https://github.com/pda/flip/blob/master/lib/flip/version.rb)
* [sinatra](https://github.com/sinatra/sinatra/blob/master/lib/sinatra/version.rb)
* [rake](https://github.com/ruby/rake/blob/master/lib/rake/version.rb)

I think the best approach for this particular PR is to override the testing task for travis until this adjustment is made.

Thoughts?

**Comment by agriffis** [comment]
Thanks for the research. I agree regarding fixing the superclass mismatch in a separate PR.

Thanks for helping out!

**Pull request merged**